lib/cpflow.rb

```diff
@@ -113,6 +113,10 @@ def self.check_cpflow_version # rubocop:disable Metrics/MethodLength
     def self.fix_help_option
       help_mappings = Thor::HELP_MAPPINGS + ["help"]
       matches = help_mappings & ARGV
+
+      # Help option works correctly for subcommands
+      return if matches && (ARGV & subcommand_names).any?
```

Comment:
I don't think we're supposed to return here. Did you test if `cpflow SUBCOMMAND COMMAND --help` works?

Reply:
Yes, it works because of `BaseSubCommand` class:
```
./cpflow terraform generate --help
```
Results in:
```
Usage:
  cpflow terraform generate

Description:
  Generates terraform configuration files based on `controlplane.yml` and `templates/` config

  cpflow terraform generate
```

Reply:
Created class method `subcommand?`